app.js

```diff
@@ -0,0 +1,17 @@
+const express = require("express");
+const app = express()
+const {getTopicData} = require("./controllers/controllerTopics")
+
+app.use(express.json())
```

Comment:
Please get rid of this line! It's not needed until you are dealing with post requests